server/docHandlers.js

```diff
@@ -0,0 +1,31 @@
+const Doc = require('doc');
+const Taboo = require('taboo');
+
+const savedoc = (req, res) => {
+  let docinst = new Doc({
+    title: req.body.name,
+    content: req.body.text,
+    owner_id: req.body.owner_id,
+    locked: false,
+    revision_id: 0,
+  });
+
+  docinst.save(function (err, doc) {
+    if (err) return console.error(err);
+      console.log(doc.name + " saved to Document collection.");
+  });     
+};
+
+const gettaboowords = (req, res) => {
```

Comment:
This does not make sense to be a function that handles a request and response. Remove this, make it a static method of the taboo schema that returns a list of all taboo words